src/components/notifications/Discord.vue

```diff
@@ -16,4 +16,36 @@
         <label for="discord-prefix-message" class="form-label">{{ $t("Prefix Custom Message") }}</label>
         <input id="discord-prefix-message" v-model="$parent.notification.discordPrefixMessage" type="text" class="form-control" autocomplete="false" :placeholder="$t('Hello @everyone is...')">
     </div>
+
+    <div class="mb-3">
+        <label for="discord-message-type" class="form-label">{{ $t("Select message type") }}</label>
+        <br>
+        <select id="discord-message-type" v-model="$parent.notification.discordChannelType" class="form-select">
+            <option value="normal">{{ $t("Normal Message") }}</option>
+            <option value="createNewForumPost">{{ $t("Create new forum post") }}</option>
+            <option value="postToThread">{{ $t("Post to existing thread") }}</option>
+        </select>
+    </div>
+
+    <div v-if="$parent.notification.discordChannelType === 'createNewForumPost'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Forum post name") }}</label>
```

Comment:
Does this create an new forum post if this does not exist?
Are there edgecases for which a helptext would be helpfull?

Reply:
It always creates a new forum post, if you want to send it to an existing forum post you need to set it to post to existing thread (this is how discord handels it, forum channel post are basicly just threads). I'm going to add a helptext.